src/fundus/publishers/de/krautreporter.py

```diff
@@ -0,0 +1,46 @@
+import json
+from datetime import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute, utility
+
+
+class KrautreporterParser(ParserProxy):
+    class V1(BaseParser):
+        _summary_selector = CSSSelector("p.article-headers-standard__teaser")
+        _subheadline_selector = CSSSelector("div.article-markdown > h2")
+        _paragraph_selector = CSSSelector("div.article-markdown > p")
+        _json_ld_selector = CSSSelector('script[type="application/ld+json"]')
+        _topic_selector = CSSSelector("div.article-headers-shared-topic")
```

Comment:
Apparently there's a different article type: Newsletter. They have a slightly different structure. [Here]( https://krautreporter.de/sinn-und-konsum/5485-jetzt-sei-gefalligst-glucklich) is an example. Also, if the parsing of the topic fails, it raises an `IndexError`, which should be caught as well